File: test_cafekiosk/src/main/java/com/testcafekiosk/spring/TestCafekioskApplication.java
package com.testcafekiosk.spring;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class TestCafekioskApplication {

    public static void main(String[] args) {
        SpringApplication.run(TestCafekioskApplication.class, args);
    }

}

File: test_cafekiosk/src/main/kotlin/com/testcafekiosk/spring/TestCafekioskApplication.kt
package com.testcafekiosk.spring

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
class TestCafekioskApplication

fun main(args: Array<String>) {
    runApplication<TestCafekioskApplication>(*args)
}

